lib/unread/reader_scopes.rb

```diff
@@ -0,0 +1,29 @@
+module Unread
+  module Reader
+    module Scopes
+      def join_read_marks(readable)
+        assert_readable(readable)
+
+        joins "LEFT JOIN #{ReadMark.table_name} as read_marks ON read_marks.readable_type = '#{readable.class.base_class.name}'
+               AND (read_marks.readable_id   = #{readable.try(readable.class.primary_key)} OR read_marks.readable_id IS NULL)
```

Comment:
We should use `send` instead of `try` here, because `readable` is asserted before.